staging/kubefed/charts/controllermanager/templates/webhook.yaml

```diff
@@ -0,0 +1,139 @@
+{{- $ca := genCA "kubefed-admission-webhook-ca" 3650 }}
```

Comment:
This file will have the same problem as CRDs. Can we apply the crd hook to ValidatingWebhookConfigurations?

Reply:
Since this may not be a problem, we'll skip worrying about it until it is.